packages/Search/test/Search_UnitTestHelpers.hpp

```diff
@@ -69,20 +68,18 @@ void checkResults( DataTransferKit::BVH<DeviceType> const &bvh,
     auto distances_host = Kokkos::create_mirror_view( distances );
     deep_copy( distances_host, distances );
 
-    TEST_COMPARE_ARRAYS( indices_host, indices_ref );
-    TEST_COMPARE_ARRAYS( offset_host, offset_ref );
-    TEST_COMPARE_FLOATING_ARRAYS( distances_host, distances_ref, 1e-14 );
+    BOOST_TEST( indices_host == indices_ref, tt::per_element() );
+    BOOST_TEST( offset_host == offset_ref, tt::per_element() );
+    // BOOST_TEST( distances_host == distances_ref, tt::per_element(),
+    // tt::tolerance( 1e-14 ) ); FIXME_BOOST
```

Comment:
This needs to be either worked around, or fixed.

Reply:
There is no real work around, you can only use one argument. What you can do is set the tolerance at the `BOOST_AUTO_CASE` level and keep the `tt::per_element()` at the `BOOST_TEST` level.